.gitignore

```diff
@@ -1,19 +1,38 @@
-*.elc
-*.a
+# OCaml files
+# see https://ocaml.org/learn/tutorials/filenames.html
 *.cm?*
-*.o
 *.opt
-*.omc
-*.run
-*~
+*.a
+*.o
+# Deprecated?
+*.annot
+
+# Ocamlbuild (used internally by Dune)
+_build/
+*.byte
+*.native
+
+# Merlin configuring files
+.merlin
+
+# Dune generated files
+*.install
+
+# Beluga debug output files
+debug.out
+
+# ELisp files
+*.elc
+
+# Mac (OS) specific files
 .DS_Store
+
+# Emacs editor setting files
+.dir-locals.el
+*~
+.projectile
+
+# ???
 .build
-.omake*
 release
-*.native
-_build
-*.byte
-*.d.native
-*.p.byte
 *.sexp
```

Comment:
Is there a generator of .sexp files?

Reply:
Yes Beluga has an s-expression dumping facility. This was for use in a past project I believe for comparing proof assistants. I don't think it's really used anymore. I would keep the entry in `.gitignore` to be safe.